src/snapshot.rs

```diff
@@ -1,12 +1,11 @@
 use crate::db_type::Result;
-use crate::{Database, DatabaseBuilder};
+use crate::{Builder, Database, Models};
 use redb::ReadableTable;
 use std::path::Path;
 
 impl Database<'_> {
-    pub fn snapshot<'a>(&self, builder: &'a DatabaseBuilder, path: &Path) -> Result<Database<'a>> {
-        // TODO: builder must have well defined models
```

Comment:
not sure, but the TODO looks like it should be preserved

Reply:
Done